src/main/java/com/mongodb/jdbc/MongoStatement.java

```diff
@@ -82,6 +85,16 @@ protected void checkClosed() throws SQLException {
         }
     }
 
+    private BsonDocument constructSQLGetResultSchemaDocument(String sql) {
+        BsonDocument command = new BsonDocument();
+        command.put("sqlGetResultSchema", new BsonInt32(1));
```

Comment:
Instead of these `new BsonInt32(1)` here and line 92, how about we update `formatVersion` to be a more general name like `BSON_ONE_INT_VALUE` and use that in place to avoid the `new` such as [here](https://github.com/mongodb/mongo-jdbc-driver/blob/2bb361d2662fa95629f3283d40ed825b2b122f08/src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java#L53).